src/main/scala/io/scalac/lab/api/endpoints4s/ApartmentsApi.scala

```diff
@@ -45,22 +44,9 @@ object ApartmentsApi extends App {
         implicit
         tupler: Tupler.Aux[U, ApiKey, I]): DocumentedEndpoint =
       super
-        .authenticatedEndpoint(request, response, docs)
+        .authenticatedEndpoint(request, response ++ unauthorized, docs)
```

Comment:
Here, I add the `unauthorized` response to the documentation of all the authenticated endpoints.

This looks like the following in the Swagger UI:

![Screenshot from 2020-10-14 21-53-16](https://user-images.githubusercontent.com/332812/96038646-b4a40300-0e67-11eb-9b64-89250d263cbc.png)